Makefile

```diff
@@ -103,6 +105,18 @@ LANTERN_YAML_PATH := installer-resources/lantern.yaml
 
 .PHONY: packages clean tun2socks android-lib android-sdk android-testbed android-debug android-release android-install docker-run
 
+define require-node
```

Comment:
We do these as macros so that they don't needless trigger reevaluation of the $(RESOURCES_DOT_GO) target.